lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -2260,7 +2271,8 @@ def main():
         lun_mappings=dict(default=[], type='list'),
         domain_mappings=dict(default=[], type='list'),
         reassign_bad_macs=dict(default=None, type='bool'),
-        boot_menu=dict(type='bool'),
+        boot_menu=dict(type='bool', default=False),
+        bios_type=dict(type='str', default='i440fx_sea_bios', choices=['i440fx_sea_bios', 'q35_ovmf', 'q35_sea_bios', 'q35_secure_boot']),
```

Comment:
the default is q35_sea_bios in 4.4.
Also, this is only applicable to x86_64.

Reply:
This PR is outdated please check https://github.com/oVirt/ovirt-ansible-collection/pull/15
We have not specified the default and kept it on the engine.